### PR TITLE
remove integration-test-with-secrets and run those tests in integration-test

### PR DESCRIPTION
**Pull request by Thanhphan1147**
### Overview

Currently we are not using operator-workflows for tests that requires secrets even though support for this has been added. We should remove the integration-tests-with-secrets job by leveraging [Add secret arguments to integration tests by yhaliaw · Pull Request #176 · canonical/operator-workflows](https://github.com/canonical/operator-workflows/pull/176/files). 

To do this we’d need to define the INTEGRATION_TEST_ARGS secret and remove the -m "not (requires_secret)" arg in the integration-tests job. Example:[Refactor integration test wfs by arturo-seijas · Pull Request #316 · canonical/indico-operator](https://github.com/canonical/indico-operator/pull/316/files)

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by nrobinaubertin — DISMISSED** [review]

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by amandahla — DISMISSED** [review]
LGTM

File: .github/workflows/integration_test.yaml [hunk 1]
```diff
@@ -10,11 +10,10 @@ jobs:
...
-      extra-arguments: >
-        -m "not (requires_secret)"
-        --openstack-rc ${GITHUB_WORKSPACE}/openrc
-        --kube-config ${GITHUB_WORKSPACE}/kube-config
-        --screenshot-dir /tmp
+      extra-arguments: >-
+        --openstack-rc=${GITHUB_WORKSPACE}/openrc
+        --kube-config=${GITHUB_WORKSPACE}/kube-config
+        --screenshot-dir=/tmp
       modules: '["test_addon", "test_core", "test_ingress", "test_cos_grafana", "test_cos_loki", "test_cos_prometheus"]'
```
Comment (weiiwang01) [inline .github/workflows/integration_test.yaml:17]: Would you please the `test_external` module here? The `test_external` module contains all the tests that require secrets.

**Review by cbartz — APPROVED** [review]
LGTM

**Review by weiiwang01 — APPROVED** [review]

**Pull request merged**